wbooks/src/app/screens/dashboard/screens/Detail/components/CommentList/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import Comment from './components/Comment';
+import CommentInput from './components/CommentInput';
+import './styles.css';
+
+const CommentList = () => {
+  const comment = {};
+  return (
+    <div className="comment-list">
+      <div className="new-comment">
+        <CommentInput />
+      </div>
+      <div className="comments">
+        <Comment {...comment} />
+        <Comment {...comment} />
+        <Comment {...comment} />
```

Comment:
Later we will see that, if we add a comment here thru an API call, this might have a `key` problem